machine.go

```diff
@@ -374,7 +378,12 @@ func (m *Machine) startup(command string, extracontent [][2]string) int {
 	w.WriteFile("/wrapper",
 		fmt.Sprintf(commandWrapper, command), 0755)
 
-	w.WriteFile("/init", initScript, 0755)
+	shebang := "#!/bin/busybox sh"
+	if mergedUsrSystem() {
+		shebang = "#!/usr/bin/busybox sh"
+	}
+	w.WriteFile("/init",
+		fmt.Sprintf("%s\n%s", shebang, initScript), 0755)
```

Comment:
Could you make the commit message explain the issue and solution a bit more.. Also for fakemachine "Build" isn't really a useful action, it would probably be start ? or correct initramfs build?